app/components/FileLoader.js

```diff
@@ -20,6 +20,8 @@ import FolderBrowser from './common/FolderBrowser';
 import PageWrapper from './PageWrapper';
 import Scroller from './common/Scroller';
 
+const shell = require('electron').shell;
```

Comment:
```suggestion
import { shell } from 'electron';
```

It's probably best to avoid mixing ES6 and require syntax.

Reply:
fair point